src/test/resources/Cucumber.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Cucumber Tests">
+
+    <test name="LoginTest" preserve-order="false">
+        <classes>
+            <class name="CucumberRunner"/>
+        </classes>
+    </test>
+</suite>
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/233101690-d175222e-ed0e-4a21-a95f-2a7b87c14ac6.png)

HINT

1)  After the Chrome 111 update, you can no longer kick off a chromedriver instance unless you add an additional chrome option:
"--remote-allow-origins=*"

 ChromeOptions options = new ChromeOptions();
        options.addArguments("--disable-notifications","--remote-allow-origins=*", "--incognito","--start-maximized");
        driver = new ChromeDriver(options);


Below are the list of available and most commonly used arguments for ChromeOptions class

**start-maximized:** Opens Chrome in maximize mode
**incognito:** Opens Chrome in incognito mode
**headless:** Opens Chrome in headless mode
**disable-extensions:** Disables existing extensions on Chrome browser
**disable-popup-blocking:** Disables pop-ups displayed on Chrome browser
**make-default-browser:** Makes Chrome default browser
**version:** Prints chrome browser version
**disable-infobars:** Prevents Chrome from displaying the notification ‘Chrome is being controlled by automated software

![image](https://user-images.githubusercontent.com/104761482/233103190-8f8aaea9-ca79-4038-b8f7-1e137c08da5a.png)

2) Test failed

![image](https://user-images.githubusercontent.com/104761482/233104377-c99063db-3663-46dd-9036-37a96fce7df8.png)

![image](https://user-images.githubusercontent.com/104761482/233104713-dc17a7dd-609a-4e2f-a91d-21f436cfec44.png)

![image](https://user-images.githubusercontent.com/104761482/233104787-c11744f5-1e21-4f5e-a3ee-0357aea4f587.png)

Reply:
I will do all this! thank you!

Reply:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/233115164-5d46464a-fa42-4bd4-89dd-05252e852a55.png)

Reply:
Alex, i made some corrections and it shows it passed on my environment? please check, i submitted again.

![image](https://user-images.githubusercontent.com/128390786/233120076-0d116e35-780a-48ea-bd51-42625fe896c2.png)

Reply:
Hello 
Your homework 27 is checked
**You’re doing great! Keep it up! Very fine!** 
💖Best wishes, the team HW (Alex,  aliaksandr@testpro.io)!💖

![giphy](https://user-images.githubusercontent.com/104761482/224489514-d22b527b-705f-4444-ab44-b5d0fded7d97.gif)


![image](https://user-images.githubusercontent.com/104761482/233121088-225f0383-369c-4790-a34b-d2776dfc8877.png)

_My recommendation:_ How to disable cucumber message

![image](https://user-images.githubusercontent.com/104761482/215353237-7823d862-662a-4d9c-a827-59bbbdc16d4c.png)

**Step 1**

![image](https://user-images.githubusercontent.com/104761482/215353276-a81d022e-3201-4295-9f8a-370159df5bcc.png)

**Step 2**  cucumber.properties and junit-platform.properties

`cucumber.publish.quiet = true`

**Step 3**

```
@CucumberOptions(
        features = {"src/test/resources/features/Login.feature"},publish = true)
```